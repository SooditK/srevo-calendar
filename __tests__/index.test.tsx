import React from "react";
import RevoCalendar from "../src/index";
import renderer from "react-test-renderer";

const testEvents = [
  {
    name: "Test Event",
    date: +new Date("November 18, 1997"),
    allDay: false,
  },
];
const testDate = new Date("November 18, 1997");

describe("RevoCalendar", () => {
  it("is truthy", () => {
    expect(RevoCalendar).toBeTruthy();
  });
});

test("Calendar component renders correctly using the passed props", () => {
  const component = renderer.create(
    <RevoCalendar
      date={testDate}
      events={testEvents}
      style={{
        borderRadius: "5px",
        border: "5px solid #4F6995",
      }}
      highlightToday={true}
      lang="en"
      primaryColor="#4F6995"
      secondaryColor="#D7E6EE"
      todayColor="#3B3966"
      textColor="#333333"
      indicatorColor="orange"
      animationSpeed={300}
      sidebarWidth={180}
      detailWidth={280}
      showDetailToggler={true}
      showSidebarToggler={true}
      onePanelAtATime={false}
      allowDeleteEvent={false}
      allowAddEvent={true}
      openDetailsOnDateSelection={true}
      timeFormat24={true}
      showAllDayLabel={false}
      detailDateFormat="DD/MM/YYYY"
    />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("Calendar renders in other languages", () => {
  const component = renderer.create(<RevoCalendar date={testDate} events={testEvents} lang="pt" />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
