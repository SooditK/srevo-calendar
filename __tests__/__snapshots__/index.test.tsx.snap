// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar component renders correctly using the passed props 1`] = `
<div
  className="sc-bdvvtL kmgMgI"
  style={
    Object {
      "border": "5px solid #4F6995",
      "borderRadius": "5px",
    }
  }
>
  <div
    className="sc-gsDKAQ ccrOKH"
    onAnimationEnd={[Function]}
  >
    <div>
      <button
        aria-label="Previous year"
        onClick={[Function]}
      >
        <svg
          aria-hidden="true"
          focusable="false"
          height="1em"
          preserveAspectRatio="xMidYMid meet"
          style={
            Object {
              "transform": "rotate(90deg)",
            }
          }
          viewBox="0 0 8 8"
          width="1em"
        >
          <path
            d="M1.5 1L0 2.5l4 4l4-4L6.5 1L4 3.5L1.5 1z"
            fill="rgb(215, 230, 238)"
          />
          <rect
            fill="rgba(0, 0, 0, 0)"
            height="8"
            width="8"
            x="0"
            y="0"
          />
        </svg>
      </button>
      <span>
        1997
      </span>
      <button
        aria-label="Next year"
        onClick={[Function]}
      >
        <svg
          aria-hidden="true"
          focusable="false"
          height="1em"
          preserveAspectRatio="xMidYMid meet"
          style={
            Object {
              "transform": "rotate(270deg)",
            }
          }
          viewBox="0 0 8 8"
          width="1em"
        >
          <path
            d="M1.5 1L0 2.5l4 4l4-4L6.5 1L4 3.5L1.5 1z"
            fill="rgb(215, 230, 238)"
          />
          <rect
            fill="rgba(0, 0, 0, 0)"
            height="8"
            width="8"
            x="0"
            y="0"
          />
        </svg>
      </button>
    </div>
    <div>
      <ul>
        <li>
          <button
            className="sc-dkPtRN qtRKj"
            onClick={[Function]}
          >
            January
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN qtRKj"
            onClick={[Function]}
          >
            February
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN qtRKj"
            onClick={[Function]}
          >
            March
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN qtRKj"
            onClick={[Function]}
          >
            April
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN qtRKj"
            onClick={[Function]}
          >
            May
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN qtRKj"
            onClick={[Function]}
          >
            June
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN qtRKj"
            onClick={[Function]}
          >
            July
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN qtRKj"
            onClick={[Function]}
          >
            August
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN qtRKj"
            onClick={[Function]}
          >
            September
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN qtRKj"
            onClick={[Function]}
          >
            October
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN dvKhjp"
            onClick={[Function]}
          >
            November
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN qtRKj"
            onClick={[Function]}
          >
            December
          </button>
        </li>
      </ul>
    </div>
  </div>
  <button
    aria-label="Toggle sidebar"
    className="sc-hKwDye dKCMjB"
    onClick={[Function]}
  >
    <svg
      height="24"
      viewBox="0 0 24 24"
      width="24"
    >
      <path
        d="M20 20h-4v-4h4v4zm-6-10h-4v4h4v-4zm6 0h-4v4h4v-4zm-12 6h-4v4h4v-4zm6 0h-4v4h4v-4zm-6-6h-4v4h4v-4zm16-8v22h-24v-22h3v1c0 1.103.897 2 2 2s2-.897 2-2v-1h10v1c0 1.103.897 2 2 2s2-.897 2-2v-1h3zm-2 6h-20v14h20v-14zm-2-7c0-.552-.447-1-1-1s-1 .448-1 1v2c0 .552.447 1 1 1s1-.448 1-1v-2zm-14 2c0 .552-.447 1-1 1s-1-.448-1-1v-2c0-.552.447-1 1-1s1 .448 1 1v2z"
        fill="rgb(215, 230, 238)"
      />
    </svg>
  </button>
  <div
    className="sc-gKclnd fnpsGh"
    onClick={[Function]}
  >
    <h1>
      November
    </h1>
    <div>
      <div>
        <div>
          SUN
        </div>
        <div>
          MON
        </div>
        <div>
          TUE
        </div>
        <div>
          WED
        </div>
        <div>
          THU
        </div>
        <div>
          FRI
        </div>
        <div>
          SAT
        </div>
      </div>
      <div>
        <div
          className="sc-eCImPb bYgdAp"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              1
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              2
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              3
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              4
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              5
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              6
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              7
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              8
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              9
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              10
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              11
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              12
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              13
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              14
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              15
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              16
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              17
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg gQbxll"
            onClick={[Function]}
          >
            <span>
              18
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              19
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              20
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              21
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              22
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              23
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              24
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              25
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              26
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              27
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              28
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              29
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              30
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-furwcr coOPkO"
    onAnimationEnd={[Function]}
  >
    <div>
      18/11/1997
      <button
        onClick={[Function]}
      >
        Add event
      </button>
    </div>
    <div>
      <div
        className="sc-iCfMLu bRDHdJ"
        onClick={[Function]}
        role="button"
      >
        <p>
          Test Event
        </p>
        <div>
          <div>
            <svg
              height="20"
              viewBox="0 0 24 24"
              width="20"
            >
              <path
                d="M12 2c5.514 0 10 4.486 10 10s-4.486 10-10 10-10-4.486-10-10 4.486-10 10-10zm0-2c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12-5.373-12-12-12zm1 12v-6h-2v8h7v-2h-5z"
                fill="rgb(79, 105, 149)"
              />
            </svg>
            <span>
              00:00
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
  <button
    aria-label="Toggle details"
    className="sc-pVTFL ma-DLr"
    onClick={[Function]}
  >
    <svg
      height="24"
      viewBox="0 0 24 24"
      width="24"
    >
      <path
        d="M24 6h-24v-4h24v4zm0 4h-24v4h24v-4zm0 8h-24v4h24v-4z"
        fill="rgb(215, 230, 238)"
      />
    </svg>
  </button>
</div>
`;

exports[`Calendar renders in other languages 1`] = `
<div
  className="sc-bdvvtL kmgMgI"
  style={Object {}}
>
  <div
    className="sc-gsDKAQ cOvxtL"
    onAnimationEnd={[Function]}
  >
    <div>
      <button
        aria-label="Ano anterior"
        onClick={[Function]}
      >
        <svg
          aria-hidden="true"
          focusable="false"
          height="1em"
          preserveAspectRatio="xMidYMid meet"
          style={
            Object {
              "transform": "rotate(90deg)",
            }
          }
          viewBox="0 0 8 8"
          width="1em"
        >
          <path
            d="M1.5 1L0 2.5l4 4l4-4L6.5 1L4 3.5L1.5 1z"
            fill="rgb(196, 220, 233)"
          />
          <rect
            fill="rgba(0, 0, 0, 0)"
            height="8"
            width="8"
            x="0"
            y="0"
          />
        </svg>
      </button>
      <span>
        1997
      </span>
      <button
        aria-label="Próximo ano"
        onClick={[Function]}
      >
        <svg
          aria-hidden="true"
          focusable="false"
          height="1em"
          preserveAspectRatio="xMidYMid meet"
          style={
            Object {
              "transform": "rotate(270deg)",
            }
          }
          viewBox="0 0 8 8"
          width="1em"
        >
          <path
            d="M1.5 1L0 2.5l4 4l4-4L6.5 1L4 3.5L1.5 1z"
            fill="rgb(196, 220, 233)"
          />
          <rect
            fill="rgba(0, 0, 0, 0)"
            height="8"
            width="8"
            x="0"
            y="0"
          />
        </svg>
      </button>
    </div>
    <div>
      <ul>
        <li>
          <button
            className="sc-dkPtRN fDOoyj"
            onClick={[Function]}
          >
            Janeiro
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN fDOoyj"
            onClick={[Function]}
          >
            Fevereiro
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN fDOoyj"
            onClick={[Function]}
          >
            Março
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN fDOoyj"
            onClick={[Function]}
          >
            Abril
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN fDOoyj"
            onClick={[Function]}
          >
            Maio
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN fDOoyj"
            onClick={[Function]}
          >
            Junho
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN fDOoyj"
            onClick={[Function]}
          >
            Julho
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN fDOoyj"
            onClick={[Function]}
          >
            Agosto
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN fDOoyj"
            onClick={[Function]}
          >
            Setembro
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN fDOoyj"
            onClick={[Function]}
          >
            Outubro
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN fXdPxx"
            onClick={[Function]}
          >
            Novembro
          </button>
        </li>
        <li>
          <button
            className="sc-dkPtRN fDOoyj"
            onClick={[Function]}
          >
            Dezembro
          </button>
        </li>
      </ul>
    </div>
  </div>
  <button
    aria-label="Alternar barra lateral"
    className="sc-hKwDye dKCMjB"
    onClick={[Function]}
  >
    <svg
      height="24"
      viewBox="0 0 24 24"
      width="24"
    >
      <path
        d="M20 20h-4v-4h4v4zm-6-10h-4v4h4v-4zm6 0h-4v4h4v-4zm-12 6h-4v4h4v-4zm6 0h-4v4h4v-4zm-6-6h-4v4h4v-4zm16-8v22h-24v-22h3v1c0 1.103.897 2 2 2s2-.897 2-2v-1h10v1c0 1.103.897 2 2 2s2-.897 2-2v-1h3zm-2 6h-20v14h20v-14zm-2-7c0-.552-.447-1-1-1s-1 .448-1 1v2c0 .552.447 1 1 1s1-.448 1-1v-2zm-14 2c0 .552-.447 1-1 1s-1-.448-1-1v-2c0-.552.447-1 1-1s1 .448 1 1v2z"
        fill="rgb(196, 220, 233)"
      />
    </svg>
  </button>
  <div
    className="sc-gKclnd eQPPLt"
    onClick={[Function]}
  >
    <h1>
      Novembro
    </h1>
    <div>
      <div>
        <div>
          DOM
        </div>
        <div>
          SEG
        </div>
        <div>
          TER
        </div>
        <div>
          QUA
        </div>
        <div>
          QUI
        </div>
        <div>
          SEX
        </div>
        <div>
          SÁB
        </div>
      </div>
      <div>
        <div
          className="sc-eCImPb bYgdAp"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              1
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              2
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              3
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              4
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              5
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              6
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              7
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              8
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              9
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              10
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              11
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              12
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              13
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              14
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              15
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              16
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              17
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg iFNhRj"
            onClick={[Function]}
          >
            <span>
              18
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              19
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              20
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              21
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              22
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              23
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              24
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              25
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              26
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              27
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              28
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              29
            </span>
          </button>
        </div>
        <div
          className="sc-eCImPb fxBoeV"
        >
          <button
            className="sc-jRQBWg licMjH"
            onClick={[Function]}
          >
            <span>
              30
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-furwcr gjjDIi"
    onAnimationEnd={[Function]}
  >
    <div>
      18/11/1997
    </div>
    <div>
      <div
        className="sc-iCfMLu hICFBH"
        onClick={[Function]}
        role="button"
      >
        <p>
          Test Event
        </p>
        <div>
          <div>
            <svg
              height="20"
              viewBox="0 0 24 24"
              width="20"
            >
              <path
                d="M12 2c5.514 0 10 4.486 10 10s-4.486 10-10 10-10-4.486-10-10 4.486-10 10-10zm0-2c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12-5.373-12-12-12zm1 12v-6h-2v8h7v-2h-5z"
                fill="rgb(79, 105, 149)"
              />
            </svg>
            <span>
              00:00
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
  <button
    aria-label="Alternar detalhes"
    className="sc-pVTFL ma-DLr"
    onClick={[Function]}
  >
    <svg
      height="24"
      viewBox="0 0 24 24"
      width="24"
    >
      <path
        d="M24 6h-24v-4h24v4zm0 4h-24v4h24v-4zm0 8h-24v4h24v-4z"
        fill="rgb(196, 220, 233)"
      />
    </svg>
  </button>
</div>
`;
